activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fix `Range#.include?` to work with Ruby 2.6 beginless and endless ranges.
```

Comment:
```suggestion
*   Fix `Range#include?` to work with beginless and endless ranges.
```

Including both Ruby 2.6 and Ruby 2.7 would make the sentence too hard to parse, and I don't think mentioning the versions that introduce the features is that important anyway.